File: tools/sap/src/test/java/top/bettercode/summer/tools/sap/cost/SapCostServiceTest.java
package top.bettercode.summer.tools.sap.cost;

import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import top.bettercode.summer.tools.sap.TestApplication;

/**
 * @author Peter Wu
 */
@SpringBootTest(classes = TestApplication.class)
class SapCostServiceTest {

  @Autowired
  SapCostService sapCostService;

  @Disabled
  @Test
  void getCosts() {
    CostResp costs = sapCostService.getCosts();
    System.err.println(costs);
  }
}

File: tools/sap/src/test/kotlin/top/bettercode/summer/tools/sap/cost/SapCostServiceTest.kt
package top.bettercode.summer.tools.sap.cost

import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest
import top.bettercode.summer.tools.sap.TestApplication

/**
 * @author Peter Wu
 */
@SpringBootTest(classes = [TestApplication::class])
internal class SapCostServiceTest {
    @Autowired
    var sapCostService: SapCostService? = null

    @Test
    @Disabled
    fun testGetCosts(): Unit {
        val costs = sapCostService!!.costs
        System.err.println(costs)
    }
}